server/matchmaker/search.py

```diff
@@ -113,7 +154,7 @@ def boundary_75(self):
 
     @property
     def failed_matching_attempts(self) -> int:
-        return self._failed_matching_attempts
+        return self._failed_matching_attempts * len(self.players)
```

Comment:
This seems very strange to me. I don't see why we would be prioritizing larger searches who have been in the queue the same amount of time as smaller searches. If some special adjustment for larger searches needs to happen then IMO it belongs in algorithm and not here.

Reply:
We are not prioritizing larger searches. I am looking at cumulated time spent in queue. At the moment four guys queueing single aquire the time bonus four times faster than if they queue up as four premades, because the bonus is only given per search. I want the time bonus per player to be always equal.

Reply:
Can you add the player count multiplier in the code that calls this. Or at the very least rename this property because IMO this behaviour is confusing:

```python3
s1 = Search([p1])
s1.register_failed_matching_attempt()
print(s1.failed_matching_attempts)
# 1

s2 = Search([p1, p2, p3, p4])
s2.register_failed_matching_attempt()
print(s2.failed_matching_attempts)
# 4

Reply:
yeah, I just realized it makes way more sense to add the multiplier when I calculate the quality